database/buildSchema/27_review.sql

```diff
@@ -0,0 +1,21 @@
+-- FUNCTION to auto-add application_id to review
+CREATE or replace FUNCTION public.review_application_id(review_id int)
+RETURNS INT AS $$
+select application_id from review_assignment where id = $1 ;
+$$ LANGUAGE SQL IMMUTABLE;
+
+-- review
+CREATE TABLE public.review (
+	id serial primary key,
+	review_assignment_id integer references public.application(id),
+	reviewer_id integer references public.user(id),
```

Comment:
Since `application_id` is computed, shouldn't `review_id` be also? (It's in review_assignment along with application_id)

Reply:
that would be pretty hard, because of stages ?

Reply:
oops ignore that

Reply:
6126eace5fb72f208823fa0d11382222fd65bd27